src/main/kotlin/app/revanced/patches/unifiprotect/localdevice/bypasswifichecks/BypassWifiChecksPatch.kt

```diff
@@ -0,0 +1,28 @@
+package app.revanced.patches.unifiprotect.localdevice.bypasswifichecks
+
+import app.revanced.patcher.data.BytecodeContext
+import app.revanced.patcher.extensions.InstructionExtensions.addInstructions
+import app.revanced.patcher.patch.BytecodePatch
+import app.revanced.patcher.patch.PatchException
+import app.revanced.patcher.patch.annotation.CompatiblePackage
+import app.revanced.patcher.patch.annotation.Patch
+import app.revanced.patches.unifiprotect.localdevice.bypasswifichecks.fingerprints.IsWifiMethodFingerprint
+
+@Patch(
+    name = "Bypass wifi checks",
+    description = "Bypasses the wifi checks that prevent the app from connecting to local devices.",
```

Comment:
What is a WIFI check?

```suggestion
    description = "Bypasses the WIFI checks that prevent the app from connecting to local devices.",
```